Solution.Versions.props

```diff
@@ -1,41 +1,44 @@
 <Project>
```

Comment:
💡 Consider filing a follow-up issue to migrate to [CentralPackageVersions](https://github.com/microsoft/MSBuildSdks/tree/main/src/CentralPackageVersions), which appears to be quite easy to adopt in non-arcade scenarios.

Reply:
* https://github.com/gitextensions/Git.hub/pull/20
* https://github.com/gitextensions/conemu-inside/pull/30
* https://github.com/gitextensions/gitextensions/pull/9161